linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/tests/wrong_multiple.fail.sh

```diff
@@ -4,5 +4,5 @@
 sed -i '/umask/d' /etc/bashrc
 echo "umask 000" >> /etc/bashrc
 echo "umask 000" >> /etc/bashrc
-echo "umask 000" >> /etc/bashrc
+echo "    [ `umask` -eq 0 ] && umask 022" >> /etc/bashrc
```

Comment:
If the problem in #11700 is that DISA content reports a fail because the regex in DISA check matches a line that doesn't have an effect on configuration because it's inside an irrelevant block of code whereas our content doesn't find it and behaves correct, why we are changing our OVAL? Shouldn't they change their OVAL instead?

Reply:
I don't think that the code block is irrelevant. It is true that the code block is executed for non login shells, at the other hand, there is no explicit mention of login shells in STIG: https://stigaview.com/products/rhel9/v1r2/RHEL-09-412055/
Why do you think the command you mention is actually irrelevant to the STIG? @jan-cerny